.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
```

Comment:
do we need `@babel/preset-react`

Reply:
No I will remove that because that is for JSX support.